profiles/rohit8020.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/76878634/193458103-5e05eea9-05b1-4690-bdb4-47f73a48f038.png",
+    "issueId": 1446,
+    "name": "Rohit Gangwar",
+    "username": "rohit8020"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```